### PR TITLE
googler: init at 3.3

### PR DESCRIPTION
**Pull request by k0ral**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Review by jb55 — APPROVED** [review]
tested + lgtm

File: pkgs/applications/misc/googler/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  propagatedBuildInputs = [ python ];
+
+  makeFlags = "PREFIX=$(out)";
+
+  meta = {
+    homepage = https://github.com/jarun/googler;
+    description = "Google Search, Google Site Search, Google News from the terminal";
+    license = stdenv.licenses.gpl3;
+    maintainers = with stdenv.maintainers; [ koral ];
+  };
```
Comment (orivej) [inline pkgs/applications/misc/googler/default.nix]: Add platforms.

Comment (k0ral) [reply pkgs/applications/misc/googler/default.nix]: Done.

**Comment by orivej** [comment]
Thank you!

**Pull request merged**